### PR TITLE
Updates for Safari 17, based on Beta 1

### PR DESCRIPTION
**Pull request by gsnedders**
This gathers results from Safari 17 on macOS Sonoma and iOS 17 Beta 1.

### CODE REVIEW

File: api/CSSCounterStyleRule.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
           "opera": "mirror",
           "opera_android": "mirror",
           "safari": {
-            "version_added": false,
+            "version_added": "17",
             "notes": "See <a href='https://webkit.org/b/167645'>bug 167645</a>."
```
Comment (Elchi3) [inline api/CSSCounterStyleRule.json:25]:
```suggestion
```

File: api/CanvasRenderingContext2D.json [hunk 2]
```diff
@@ -815,7 +815,7 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": false,
+                "version_added": "17",
                 "notes": "See <a href='https://webkit.org/b/226964'>bug 226964</a>."
```
Comment (Elchi3) [inline api/CanvasRenderingContext2D.json:819]:
```suggestion
```

**Review by Elchi3 — COMMENTED** [review]
Thanks! Please run `npm run fix` to set `experimental` status correctly. Also, I think implementation bugs in notes can be removed now.

File: css/properties/list-style.json [hunk 3]
```diff
@@ -61,7 +61,7 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": false
+                "version_added": "17"
```
Comment (nt1m) [inline css/properties/list-style.json:64]: Safari 17 doesn't support symbols() yet.

**Pull request merged**